src/guest/vm_flash.cc

```diff
@@ -22,6 +22,7 @@
 namespace vm_manager {
 
 constexpr const char *kVirtualUsbDiskPath("/tmp/flash.vfat");
+std::string kVirtualDisk;
```

Comment:
We'd better avoid global variable except const expression. and the k******* means a const variable.